webapp/src/utils/index.ts

```diff
@@ -0,0 +1,11 @@
+import {id as pluginId} from '../manifest';
+
+const getBaseUrls = (): {pluginApiBaseUrl: string} => {
+    const url = new URL(window.location.href);
```

Comment:
We can use the URL from redux here. once [this](https://github.com/Brightscout/mattermost-plugin-template/pull/9) PR is merged. Please add a TODO comment for the same